src/ImageGallery.jsx

```diff
@@ -87,6 +87,7 @@ export default class ImageGallery extends React.Component {
     stopPropagation: PropTypes.bool,
     additionalClass: PropTypes.string,
     useTranslate3D: PropTypes.bool,
+    multiSlide: PropTypes.bool,
```

Comment:
Would be cool if we could refactor this to be a number, and make it more dynamic in how many slides it loads.